ofl/notoserifdogra/OFL.txt

```diff
@@ -1,30 +1,29 @@
-Copyright 2012 Google Inc. All Rights Reserved.
-
-This Font Software is licensed under the SIL Open Font License,
-Version 1.1.
+Copyright 2022 The Noto Project Authors (https://github.com/notofonts/noto-project-template)
```

Comment:
@simoncozens is this change of copyright string okay?

Reply:
Yes, this is correct. (see Dave at https://github.com/googlefonts/fontbakery/pull/3681#issuecomment-1081954210)

Reply:
Although, urgh, it should have customised the URL.